base_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './base_decorator'
+
+class CapitalizeDecorator < BaseDecorator
```

Comment:
# Base Decoiartor
- Decorator class inherits from Namable.
- The constructor a namable object from params is assigned to an instance variable.
- **Also make sure your Decoration test works fine after updating the class**